tests.py

```diff
@@ -1493,6 +1547,43 @@ def test_update_time(self, mock_create_or_update):
                                                  "times", False)
 
     @patch("pymesync.TimeSync._create_or_update")
+    def test_create_time_with_string_duration(self, mock_create_or_update):
+        """Tests that TimeSync._create_time will convert a string duration to
+        the correct number of seconds"""
+        params = {
+            "duration": "3h30m",
+            "project": "ganeti-web-manager",
+            "user": "example-user",
+            "activities": ["documenting"],
+            "notes": "Worked on docs",
+            "issue_uri": "https://github.com/",
+            "date_worked": "2014-04-17",
+        }
+
+        self.ts.create_time(params)
+
+        mock_create_or_update.assert_called_with(params, None, "time", "times")
```

Comment:
This is testing that `create_time` does what it is supposed to do currently, since you are asserting that `_create_or_update` was called with the unchanged parameter dictionary.